scipy/stats/stats.py

```diff
@@ -197,14 +197,21 @@
            'pearsonr', 'fisher_exact', 'spearmanr', 'pointbiserialr',
            'kendalltau', 'linregress', 'theilslopes', 'ttest_1samp',
            'ttest_ind', 'ttest_ind_from_stats', 'ttest_rel', 'kstest',
-           'chisquare', 'power_divergence', 'ks_2samp', 'mannwhitneyu',
+           'chisquare', 'power_divergence', 'ks_2samp',
            'tiecorrect', 'ranksums', 'kruskal', 'friedmanchisquare',
-           'chisqprob', 'betai',
+           'chisqprob', 'betai', 'mww',
            'f_value_wilks_lambda', 'f_value', 'f_value_multivariate',
            'ss', 'square_of_sums', 'fastsort', 'rankdata', 'nanmean',
            'nanstd', 'nanmedian', 'combine_pvalues', ]
 
 
+# This class will be used by place_poles to return its results
```

Comment:
This comment line can be removed; ``place_poles`` is an unrelated function in ``scipy.signal``. Keep the link to the activestate recipe though.